src/Interfaces/AbstractPolynomialZonotope.jl

```diff
@@ -21,3 +21,7 @@ abstract type AbstractPolynomialZonotope{N} <: LazySet{N} end
 
 isconvextype(::Type{<:AbstractPolynomialZonotope}) = false
 isboundedtype(::Type{<:AbstractPolynomialZonotope}) = true
+
+function Base.:*(M::Union{Real, AbstractMatrix, AbstractVector, LinearAlgebra.UniformScaling}, P::AbstractPolynomialZonotope)
+    return linear_map(M, P)
+end
```

Comment:
`*` is lazy for all other set types., so this would make the library inconsistent.

Reply:
Should I simpli remove the operator overloading?

I guess this will be automatically fixed once the signatures for linear_map are updated `ConvexSet --> LazySet` and SPZ is made a subtype of LazySet?

Reply:
After that change it will become `LinearMap` (the lazy operation).

Reply:
Yes, please remove this method.